tests/data_context/cloud_data_context/test_expectation_suite_crud.py

```diff
@@ -134,3 +199,264 @@ def test_list_expectation_suites(
             resource_name=suite_name_2,
         ),
     ]
+
+
+@pytest.mark.unit
+@pytest.mark.cloud
+def test_create_expectation_suite_saves_suite_to_cloud(
+    empty_base_data_context_in_cloud_mode: BaseDataContext,
+    mocked_post_response: Callable[[], MockResponse],
+) -> None:
+    context = empty_base_data_context_in_cloud_mode
+
+    suite_name = "my_suite"
+    existing_suite_names = []
+
+    with mock.patch(
+        "great_expectations.data_context.data_context.cloud_data_context.CloudDataContext.list_expectation_suite_names",
+        return_value=existing_suite_names,
+    ), mock.patch("requests.post", autospec=True, side_effect=mocked_post_response):
+        suite = context.create_expectation_suite(suite_name)
+
+    assert suite.ge_cloud_id is not None
+
+
+@pytest.mark.unit
+@pytest.mark.cloud
+def test_create_expectation_suite_overwrites_existing_suite(
+    empty_base_data_context_in_cloud_mode: BaseDataContext,
+    mocked_post_response: Callable[[], MockResponse],
+) -> None:
+    context = empty_base_data_context_in_cloud_mode
+
+    suite_name = "my_suite"
+    existing_suite_names = [suite_name]
+
+    with mock.patch(
+        "great_expectations.data_context.data_context.cloud_data_context.CloudDataContext.list_expectation_suite_names",
+        return_value=existing_suite_names,
+    ), mock.patch("requests.post", autospec=True, side_effect=mocked_post_response):
+        suite = context.create_expectation_suite(suite_name, overwrite_existing=True)
+
+    assert suite.ge_cloud_id is not None
+
+
+@pytest.mark.unit
+@pytest.mark.cloud
+def test_create_expectation_suite_namespace_collision_raises_error(
+    empty_base_data_context_in_cloud_mode: BaseDataContext,
+) -> None:
+    context = empty_base_data_context_in_cloud_mode
+
+    suite_name = "my_suite"
+    existing_suite_names = [suite_name]
+    with mock.patch(
+        "great_expectations.data_context.data_context.cloud_data_context.CloudDataContext.list_expectation_suite_names",
+        return_value=existing_suite_names,
```

Comment:
`nicetohave`
Could you move these repeated mocks into a well-named fixture?

Reply:
sure! mind if I do that in a follow-up PR?